src/EnergyPlus/FluidCoolers.cc

```diff
@@ -715,7 +718,8 @@ namespace FluidCoolers {
                 }
                 ShowContinueError(
                     "Design fluid cooler UA field must be left blank when nominal fluid cooler capacity performance input method is used.");
-                ErrorsFound = true;
+                ShowContinueError("Design fluid cooler UA value will be reset to zero and the simulation continuous.");
+                SimpleFluidCooler(FluidCoolerNum).HighSpeedFluidCoolerUA = 0.0;
```

Comment:
@Nigusse 

Since you are now allowing this code to continue without issuing a fatal, the severe error a couple lines above should be converted to a warning.

Reply:
Changed the Severe Error message to Warning Error message.